lib/BackgroundJob/Tasks/CreateClustersTask.php

```diff
@@ -208,8 +207,9 @@ private function createClusterIfNeeded(string $userId) {
 		}
 
 		// Prevents not create/recreate the clusters unnecessarily.
-		$this->config->setUserValue($userId, 'facerecognition', 'recreate-clusters', 'false');
-		$this->config->setUserValue($userId, 'facerecognition', 'force-create-clusters', 'false');
+
+		$this->settingsService->setNeedRecreateClusters(false, $userId);
+		$this->settingsService->setForceCreateClusters(false, $userId);
```

Comment:
@stalker314314 

I am doing a literal migration ... Line by line to not modify the logic ..
But now I observe that we have differentiated these flags, but  'force-create-clusters' never is reseted.. This is the only place where it is assigned. There is no other line that sets it to true... :confused: 

Then I try to follow the commits to see when this code was last edited.

Reply:
Oh, I used this as a helper for tests. There is comment about this:
https://github.com/matiasdelellis/facerecognition/blob/4a9e6542009d8efc9bae1a3ab0599cf59d0f3a8d/lib/BackgroundJob/Tasks/CreateClustersTask.php#L163

and it used in one test only, to force recreation:
https://github.com/matiasdelellis/facerecognition/search?q=force-create-clusters&unscoped_q=force-create-clusters

We did have it, I think, in code from users, but it was not needed at one point.

Reply:
Ohh.. Now remember that. Thank you!.. 😄

Reply:
But, to be honest, this solution sucks. I cannot remember why I didn't opt for some proper test mocks, but for full config value. Not sure if it was just easier/lack of time/patience, or there were some problems:/ But, looking from this distance, it is ugly:)

Reply:
haha.. For now do not worry .. :wink: 
Well. Talking about testing, I have to fix them again .. In this PR I broke up absolutely everyone .. :sweat_smile: